libsolidity/ast/AST.h

```diff
@@ -195,6 +196,8 @@ class PragmaDirective: public ASTNode
 	std::vector<Token::Value> const& tokens() const { return m_tokens; }
 	std::vector<ASTString> const& literals() const { return m_literals; }
 
+	SemVerMatchExpression parseSemVer() const;
```

Comment:
It is quite dangerous to expose this as a public function because the internal use is preceded by some checks against the length of `tokens()`